examples/dockerfile/buildx.bzl

```diff
@@ -14,48 +148,93 @@ def _impl_configure_buildx(rctx):
     compatible_with = "[]"
     builder_name = "builder-docker"
     if has_docker:
-        # TODO(alex): a less hacky way for a repo rule to find this transitive repo label?
-        if repo_utils.platform(rctx) == "darwin_amd64":
-            multitool_label = "@@rules_multitool~~multitool~multitool.macos_x86_64//tools/buildx:macos_x86_64_executable"
```

Comment:
@alexeagle sorry for removing your multitool setup from this example but i need the buildx.bzl file to be standalone and only source that needs to be copied over.

Reply:
haha, that's okay if it's getting in the way